packages/live-status-gateway/src/wsHandler.ts

```diff
@@ -71,108 +99,226 @@ export interface WebSocketTopic {
 	sendMessage(ws: WebSocket, msg: object): void
 }
 
-export type ObserverForCollection<T> = T extends keyof CorelibPubSubCollections
-	? Observer<CorelibPubSubCollections[T]>
-	: undefined
+const DEFAULT_THROTTLE_PERIOD_MS = 20
 
-export abstract class CollectionBase<
-	T,
-	TPubSub extends CorelibPubSub | undefined,
-	TCollection extends keyof CorelibPubSubCollections
-> {
+export abstract class CollectionBase<T, TCollection extends keyof CorelibPubSubCollections> {
```

Comment:
I realize this is a refactoring of the older class, but considering `PublicationCollection` has now been added to this file, do you think it would make sense to move it to it's own file, to avoid it growing?